tests/integration/components/linkable-header-test.js

```diff
@@ -0,0 +1,24 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module('Integration | Component | linkable-header', function(hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders', async function(assert) {
```

Comment:
Some additional testing could be helpful, such as checking that an anchor is made based on the link target.

Note, the test may need `this.owner.setupRouter();` for the anchor links to have anchors within a test.

Reply:
This may be an area where I would like to pair or have you submit a commit.

Reply:
Wonderful! Great job~